book/src/datatypes.md

```diff
@@ -1 +1,17 @@
 # Data types
+
+
+## Dictionary, array, set
```

Comment:
Maybe this can be merged into https://github.com/0xPARC/pod2/blob/main/book/src/values.md ?
And replace the `MerkleTree` with these three?

Reply:
Right makes sense, will move it there.